### PR TITLE
add transducer_stateless with char unit to AIShell

### PR DESCRIPTION
**Pull request by pingfengluo**
1. add transducer_stateless with char unit to AIShell
2.
||test|
|--|--|
|CER| 5.7% |

### CODE REVIEW

**Comment by pingfengluo** [comment]
@csukuangfj ready for review, thanks.

File: egs/aishell/ASR/RESULTS.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
 ## Results
 
+### Aishell training results (Transducer-stateless)
```
Comment (csukuangfj) [inline egs/aishell/ASR/RESULTS.md:3]: Could you also update `README.md` in the root directory of the repo to add results for this model?

**Comment by csukuangfj** [comment]
Thanks. Looks great to me. Just left a minor comment.

Could you also upload the pre-trained model to hugging face and put the link to it either in the comment of this PR or in RESULTS.md?
Please see
- https://huggingface.co/csukuangfj/icefall-asr-librispeech-transducer-stateless-bpe-500-2021-12-27
- https://github.com/k2-fsa/icefall/pull/152#issuecomment-1000041574
for examples.

**Comment by pingfengluo** [comment]
ok. updated.

**Pull request merged**